test-fixtures/generate-metrics.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+# set -x # For debug
+set -e
+
+BUILD_LOG_FILE="$1"
+TYPE_LOG_FILE="$2"
+THREESHOLD_UNIT_TEST=310
```

Comment:
As we start working to reduce the warnings, we can reduce this number. Same for the line below

Reply:
Ok perfect! So when we fix warnings we can change it directly in this file 💪